centos_ci.sh

```diff
@@ -345,7 +345,8 @@ function perform_release() {
 }
 
 function perform_pr() {
-  make prerelease synopsis_docs link_check_docs
+  #make prerelease synopsis_docs link_check_docs
+  env
   MINISHIFT_VM_DRIVER=kvm make integration_pr
```

Comment:
@agajdosi Could you also apply the following patch to this file.. and use these env variables in the feature file
```diff
 centos_ci.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/centos_ci.sh b/centos_ci.sh
index 3e707a912..8652539a5 100644
--- a/centos_ci.sh
+++ b/centos_ci.sh
@@ -29,7 +29,7 @@ LATEST="latest"
 function load_jenkins_vars() {
   if [ -e "jenkins-env" ]; then
     cat jenkins-env \
-      | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId|GH_TOKEN|CICO_API_KEY|API_TOKEN|JOB_NAME|RELEASE_VERSION|GITHUB_TOKEN|REPO|BRANCH|SKIP_INTEGRATION_TEST)=" \
+      | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId|GH_TOKEN|CICO_API_KEY|API_TOKEN|JOB_NAME|RELEASE_VERSION|GITHUB_TOKEN|REPO|BRANCH|SKIP_INTEGRATION_TEST|RH_REGISTRY_USERNAME|RH_REGISTRY_PASSWORD)=" \
       | sed 's/^/export /g' \
       > ~/.jenkins-env
     source ~/.jenkins-env
```

Reply:
Hi @anjannath, sure, will do that. Thank you for figuring this out :1st_place_medal: